zsysconf_values_linux_loongarch64.go

```diff
@@ -0,0 +1,111 @@
+// Code generated by cmd/cgo -godefs; DO NOT EDIT.
+// cgo -godefs sysconf_values_linux.go
+
```

Comment:
Also, we'll need proper `// +build` and `//go:build` lines here (depending on the `GOARCH` being chosen) in order to fix the build with Go versions which don't know about that yet.

Reply:
good idea, `//+build` is a good way  at the moment. As for the `GOARCH` issue, just wait for their decision.